<a href="https://colab.research.google.com/github/NatanRod99/TratamentosDadosEnem/blob/main/Enem_tratamento_de_dados_2020.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
import numpy as np
import pandas as pd

In [ ]:
dados = pd.read_csv('/microdados_enem_2019_sp.csv', sep= ';', encoding='iso-8859-1')
# "encoding:" é uma codificação de caracteres

In [ ]:
dados.head()

NU_INSCRICAO  CO_MUNICIPIO_RESIDENCIA  ... NU_NOTA_REDACAO  Q025
0     1,90E+11                  3546801  ...           940.0     B
1     1,90E+11                  3538204  ...             NaN     A
2     1,90E+11                  3550308  ...           880.0     A
3     1,90E+11                  3548708  ...           560.0     B
4     1,90E+11                  3549409  ...             NaN     B

[5 rows x 44 columns]

In [ ]:
# O número total de inscritos em São Paulo foi de 813772, sendo de 15,97% de todo o Brasil que foi de 5095308 inscritos.
dados.shape

(813772, 44)

In [ ]:
# Análise dos tipos de atributos
# Object: Strings
# Int64: Inteiros
# Float64: Reais
# Complex: Complesxos
dados.dtypes

NU_INSCRICAO                object
CO_MUNICIPIO_RESIDENCIA      int64
NO_MUNICIPIO_RESIDENCIA     object
CO_UF_RESIDENCIA             int64
SG_UF_RESIDENCIA            object
NU_IDADE                     int64
TP_SEXO                     object
TP_ESTADO_CIVIL              int64
TP_COR_RACA                  int64
TP_NACIONALIDADE             int64
CO_MUNICIPIO_NASCIMENTO    float64
NO_MUNICIPIO_NASCIMENTO     object
CO_UF_NASCIMENTO           float64
SG_UF_NASCIMENTO            object
TP_ST_CONCLUSAO              int64
TP_ANO_CONCLUIU              int64
TP_ESCOLA                    int64
TP_ENSINO                  float64
IN_TREINEIRO                 int64
CO_ESCOLA                  float64
CO_MUNICIPIO_ESC           float64
NO_MUNICIPIO_ESC            object
CO_UF_ESC                  float64
SG_UF_ESC                   object
TP_DEPENDENCIA_ADM_ESC     float64
TP_LOCALIZACAO_ESC         float64
TP_SIT_FUNC_ESC            float64
TP_PRESENCA_CN               int64
TP_PRESENCA_CH      

In [ ]:
dados1 = dados.drop(columns=['CO_MUNICIPIO_RESIDENCIA'])

In [ ]:
dados1.head()

NU_INSCRICAO NO_MUNICIPIO_RESIDENCIA  ...  NU_NOTA_REDACAO Q025
0     1,90E+11            Santa Isabel  ...            940.0    B
1     1,90E+11             Pinhalzinho  ...              NaN    A
2     1,90E+11               São Paulo  ...            880.0    A
3     1,90E+11   São Bernardo do Campo  ...            560.0    B
4     1,90E+11    São Joaquim da Barra  ...              NaN    B

[5 rows x 43 columns]

In [ ]:
dados1 = dados1.drop(columns=['CO_UF_RESIDENCIA','SG_UF_RESIDENCIA','CO_MUNICIPIO_NASCIMENTO'],)

In [ ]:
dados1.head()

NU_INSCRICAO NO_MUNICIPIO_RESIDENCIA  ...  NU_NOTA_REDACAO Q025
0     1,90E+11            Santa Isabel  ...            940.0    B
1     1,90E+11             Pinhalzinho  ...              NaN    A
2     1,90E+11               São Paulo  ...            880.0    A
3     1,90E+11   São Bernardo do Campo  ...            560.0    B
4     1,90E+11    São Joaquim da Barra  ...              NaN    B

[5 rows x 40 columns]

In [ ]:
dados1 = dados1.drop(columns=['NO_MUNICIPIO_NASCIMENTO','CO_UF_NASCIMENTO','SG_UF_NASCIMENTO','TP_ANO_CONCLUIU','TP_ENSINO','CO_MUNICIPIO_ESC','CO_UF_ESC','SG_UF_ESC'],)
# Limpeza

In [ ]:
dados1.head()

NU_INSCRICAO NO_MUNICIPIO_RESIDENCIA  ...  NU_NOTA_REDACAO Q025
0     1,90E+11            Santa Isabel  ...            940.0    B
1     1,90E+11             Pinhalzinho  ...              NaN    A
2     1,90E+11               São Paulo  ...            880.0    A
3     1,90E+11   São Bernardo do Campo  ...            560.0    B
4     1,90E+11    São Joaquim da Barra  ...              NaN    B

[5 rows x 32 columns]

In [ ]:
dados1 = dados1.drop(columns=['TP_DEPENDENCIA_ADM_ESC','TP_LOCALIZACAO_ESC','TP_SIT_FUNC_ESC'],)

In [ ]:
dados1.head()

NU_INSCRICAO NO_MUNICIPIO_RESIDENCIA  ...  NU_NOTA_REDACAO Q025
0     1,90E+11            Santa Isabel  ...            940.0    B
1     1,90E+11             Pinhalzinho  ...              NaN    A
2     1,90E+11               São Paulo  ...            880.0    A
3     1,90E+11   São Bernardo do Campo  ...            560.0    B
4     1,90E+11    São Joaquim da Barra  ...              NaN    B

[5 rows x 29 columns]

In [ ]:
dados1.shape

(813772, 29)

In [ ]:
# Corrigindo erros de notas
dados1.loc[:,'NU_NOTA_CN'] /= 10

In [ ]:
dados1.head(12)

NU_INSCRICAO NO_MUNICIPIO_RESIDENCIA  ...  NU_NOTA_REDACAO Q025
0      1,90E+11            Santa Isabel  ...            940.0    B
1      1,90E+11             Pinhalzinho  ...              NaN    A
2      1,90E+11               São Paulo  ...            880.0    A
3      1,90E+11   São Bernardo do Campo  ...            560.0    B
4      1,90E+11    São Joaquim da Barra  ...              NaN    B
5      1,90E+11              Embu-Guaçu  ...            560.0    B
6      1,90E+11     São José dos Campos  ...              NaN    A
7      1,90E+11     São José dos Campos  ...              NaN    B
8      1,90E+11               Guarulhos  ...            880.0    B
9      1,90E+11               São Paulo  ...              NaN    B
10     1,90E+11   São José do Rio Preto  ...            580.0    A
11     1,90E+11                  Sumaré  ...              NaN    B

[12 rows x 29 columns]

In [ ]:
dados1.loc[:,'NU_NOTA_CH'] /= 10
dados1.loc[:,'NU_NOTA_LC'] /= 10
dados1.loc[:,'NU_NOTA_MT'] /= 10

In [ ]:
dados1.head()

NU_INSCRICAO NO_MUNICIPIO_RESIDENCIA  ...  NU_NOTA_REDACAO Q025
0     1,90E+11            Santa Isabel  ...            940.0    B
1     1,90E+11             Pinhalzinho  ...              NaN    A
2     1,90E+11               São Paulo  ...            880.0    A
3     1,90E+11   São Bernardo do Campo  ...            560.0    B
4     1,90E+11    São Joaquim da Barra  ...              NaN    B

[5 rows x 29 columns]

In [ ]:
# Renomeando variáveis(Colunas)
dados1 = dados1.rename(columns={'NU_NOTA_REDACAO': 'NOTA_REDACAO'})

In [ ]:
dados1.head()

NU_INSCRICAO NO_MUNICIPIO_RESIDENCIA  ...  NOTA_REDACAO Q025
0     1,90E+11            Santa Isabel  ...         940.0    B
1     1,90E+11             Pinhalzinho  ...           NaN    A
2     1,90E+11               São Paulo  ...         880.0    A
3     1,90E+11   São Bernardo do Campo  ...         560.0    B
4     1,90E+11    São Joaquim da Barra  ...           NaN    B

[5 rows x 29 columns]

In [ ]:
dados1 = dados1.rename(columns={'NU_NOTA_CN': 'NOTA_CN','NU_NOTA_CH': 'NOTA_CH','NU_NOTA_LC': 'NOTA_LC','NU_NOTA_MT': 'NOTA_MT'})

In [ ]:
dados1 = dados1.rename(columns={'NU_NOTA_COMP1': 'COMP1','NU_NOTA_COMP2': 'COMP2','NU_NOTA_COMP3': 'COMP3','NU_NOTA_COMP4': 'COMP4','NU_NOTA_COMP5':'COMP5'})

In [ ]:
dados1 = dados1.rename(columns={'NU_IDADE': 'IDADE','TP_SEXO': 'SEXO','TP_COR_RACA': 'RACA','Q025': 'INTERNET','TP_ESCOLA':'ESCOLA'})

In [ ]:
dados1["RACA"] = dados1["RACA"].replace({0:"nao_declarado", 1:"branca", 2:"preta", 3:"parda", 4:"amarela", 5:"indigena"})

In [ ]:
dados1.head()

NU_INSCRICAO NO_MUNICIPIO_RESIDENCIA  IDADE  ...  COMP5  NOTA_REDACAO INTERNET
0     1,90E+11            Santa Isabel     22  ...  200.0         940.0        B
1     1,90E+11             Pinhalzinho     19  ...    NaN           NaN        A
2     1,90E+11               São Paulo     17  ...  200.0         880.0        A
3     1,90E+11   São Bernardo do Campo     19  ...  100.0         560.0        B
4     1,90E+11    São Joaquim da Barra     37  ...    NaN           NaN        B

[5 rows x 29 columns]

In [ ]:
# OUTRA MANEIRA
# dados1['RACA'] = dados1['RACA'].replace([0,1,2,3,4,5], ['Não Declarado','Branca','Preta','Parda','Amarela','Indígena'])

In [ ]:
dados1["TP_LINGUA"] = dados1["TP_LINGUA"].replace({0:"Inglês", 1:"Espanhol"})

In [ ]:
dados1["ESCOLA"] = dados1["ESCOLA"].replace({1:"não_respondeu", 2:"pública", 3:"privada", 4:"exterior"})

In [ ]:

dados1["INTERNET"] = dados1["INTERNET"].replace({"A":"Não", "B":"Sim"})


***ANÁLISE EXPLORATÓRIA E TRANFORMAÇÃO DOS DADOS***

In [ ]:
# ANÁLISE DAS IDADES
dados1['IDADE'].value_counts()

17    219313
18    147548
19     81236
16     59316
20     53009
       ...  
86         1
4          1
3          1
91         1
2          1
Name: IDADE, Length: 80, dtype: int64

In [ ]:
dados1['IDADE'].value_counts().sort_index()

2     1
3     1
4     1
5     2
6     2
     ..
80    2
81    3
82    2
86    1
91    1
Name: IDADE, Length: 80, dtype: int64

In [ ]:
#Alguns valores estranhos e curiosos: idades menores que 12 anos e maiores de 80 anos.
menores_12 = dados1.query('IDADE < 12')['NO_MUNICIPIO_RESIDENCIA'].value_counts()

In [ ]:
# Inscrições com idade abaixo de 12 anos serão excluídos devido a alta probabilidade de erro.
menores_12

São Paulo                 5
Tambaú                    1
Ribeirão Preto            1
Bálsamo                   1
São Bernardo do Campo     1
Embu das Artes            1
Queiroz                   1
Santana de Parnaíba       1
Vargem Grande Paulista    1
Auriflama                 1
Araçatuba                 1
Campinas                  1
Name: NO_MUNICIPIO_RESIDENCIA, dtype: int64

In [ ]:
dados_maiores_11 = dados1.loc[dados1.IDADE > 11]

In [ ]:
dados_maiores_11.head()

NU_INSCRICAO NO_MUNICIPIO_RESIDENCIA  IDADE  ...  COMP5  NOTA_REDACAO INTERNET
0     1,90E+11            Santa Isabel     22  ...  200.0         940.0      Sim
1     1,90E+11             Pinhalzinho     19  ...    NaN           NaN      Não
2     1,90E+11               São Paulo     17  ...  200.0         880.0      Não
3     1,90E+11   São Bernardo do Campo     19  ...  100.0         560.0      Sim
4     1,90E+11    São Joaquim da Barra     37  ...    NaN           NaN      Sim

[5 rows x 29 columns]

In [ ]:
dados_maiores_11['IDADE'].value_counts().sort_index()

12        6
13       29
14      355
15    11583
16    59316
      ...  
80        2
81        3
82        2
86        1
91        1
Name: IDADE, Length: 73, dtype: int64

***TREINEIROS***

In [ ]:
dados_maiores_11.query('IN_TREINEIRO == 1')['IN_TREINEIRO'].value_counts()

1    93988
Name: IN_TREINEIRO, dtype: int64

In [ ]:
treineiros = dados_maiores_11.loc[dados_maiores_11.IN_TREINEIRO == 1]

In [ ]:
treineiros.head()

NU_INSCRICAO NO_MUNICIPIO_RESIDENCIA  IDADE  ... COMP5  NOTA_REDACAO INTERNET
20      1,90E+11                  Santos     15  ...  80.0         560.0      Sim
21      1,90E+11                Panorama     16  ...   NaN           NaN      Não
106     1,90E+11               São Paulo     16  ...  80.0         540.0      Sim
109     1,90E+11        Águas de Lindóia     16  ...   NaN           NaN      Sim
159     1,90E+11        Águas de Lindóia     15  ...   NaN           NaN      Sim

[5 rows x 29 columns]

In [ ]:
treineiros.to_csv('treineiros_enem_2019_sp.csv', encoding = 'iso-8859-1', index = False)

In [ ]:
# Separação entre os treineiros e os vestibulandos
vestibulandos = dados_maiores_11.loc[dados_maiores_11.IN_TREINEIRO == 0]

In [ ]:
vestibulandos.head()

NU_INSCRICAO NO_MUNICIPIO_RESIDENCIA  IDADE  ...  COMP5  NOTA_REDACAO INTERNET
0     1,90E+11            Santa Isabel     22  ...  200.0         940.0      Sim
1     1,90E+11             Pinhalzinho     19  ...    NaN           NaN      Não
2     1,90E+11               São Paulo     17  ...  200.0         880.0      Não
3     1,90E+11   São Bernardo do Campo     19  ...  100.0         560.0      Sim
4     1,90E+11    São Joaquim da Barra     37  ...    NaN           NaN      Sim

[5 rows x 29 columns]

In [ ]:
vestibulandos.shape

(719768, 29)

In [ ]:
#Análise das presenças
# Ciências da Natureza
vestibulandos['TP_PRESENCA_CN'].value_counts()

1    500805
0    218693
2       270
Name: TP_PRESENCA_CN, dtype: int64

In [ ]:
vestibulandos['TP_PRESENCA_CH'].value_counts()
# Ciências Humanas

1    533142
0    185969
2       657
Name: TP_PRESENCA_CH, dtype: int64

In [ ]:
vestibulandos['TP_PRESENCA_LC'].value_counts()
# Linguagens e códigos

1    533142
0    185969
2       657
Name: TP_PRESENCA_LC, dtype: int64

In [ ]:
vestibulandos['TP_PRESENCA_MT'].value_counts()
# Matemática

1    500805
0    218693
2       270
Name: TP_PRESENCA_MT, dtype: int64

# RESUMO DA PRESENÇA NAS PROVAS:

# Ciências da Natureza: 500805 presentes, 218693 faltaram e 270 eliminados.

# Ciências Humanas: 533142 presentes, 185969 faltaram e 657 eliminados.

# Linguagens e Códigos: 533142 presentes, 185969 faltaram e 657 eliminados.

# Matemática: 500805 presentes, 218693 faltaram e 270 eliminados.

In [ ]:
vestibulandos['TP_STATUS_REDACAO'].value_counts().sort_index()
# Componentes da redação

1.0    519763
2.0       913
3.0      1417
4.0      5513
6.0      3626
7.0       543
8.0       828
9.0       539
Name: TP_STATUS_REDACAO, dtype: int64

In [ ]:
# Valores Ausentes
vestibulandos['IDADE'].isnull().sum()
# Nenhum problema com relação a preenchimento

0

In [ ]:
provas = ['NOTA_REDACAO','NOTA_CN','NOTA_MT','NOTA_LC','NOTA_CH']
# Valores ausentes

In [ ]:
# Registros NAN nas notas
vestibulandos[provas].isnull().sum()

NOTA_REDACAO    186626
NOTA_CN         218963
NOTA_MT         218963
NOTA_LC         186626
NOTA_CH         186626
dtype: int64

In [ ]:
presenca_total = ['TP_PRESENCA_CN','TP_PRESENCA_CH','TP_PRESENCA_MT','TP_PRESENCA_LC']
# Presença de cada uma das disciplinas

In [ ]:
#Criado uma coluna que soma as presenças nas provas
vestibulandos['presenca_total'] = vestibulandos[presenca_total].sum(axis=1)

/usr/local/lib/python3.7/dist-packages/ipykernel_launcher.py:2: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  


In [ ]:
vestibulandos.head()
# Foi criado uma coluna a mais - Presença Total
# Faz uma separação doa alunos que foram, que faltaram...

NU_INSCRICAO NO_MUNICIPIO_RESIDENCIA  ...  INTERNET presenca_total
0     1,90E+11            Santa Isabel  ...       Sim              4
1     1,90E+11             Pinhalzinho  ...       Não              0
2     1,90E+11               São Paulo  ...       Não              4
3     1,90E+11   São Bernardo do Campo  ...       Sim              4
4     1,90E+11    São Joaquim da Barra  ...       Sim              0

[5 rows x 30 columns]

In [ ]:
vestibulandos['presenca_total'].value_counts().sort_index()
# Se a soma for 0 faltou nos dois dias
# Se a soma for 2, foi em um dia
# Se a soma for 4, foi nos dois dias
# Se a soma for 6 ou 8 teve desclassificação.

0    185175
2     33726
4    500526
6       341
Name: presenca_total, dtype: int64

# COMPARECIMENTO NOS DIAS DAS PROVAS:

# 500526 inscritos compareceram nos dois dias.

# 185175 inscritos faltaram nos dois dias.

# 33726 inscritos faltaram em um dos dias.

# 341 foram desclassificados.

In [ ]:
vestibulandos_presentes = vestibulandos.loc[vestibulandos.presenca_total == 4]
# Aqueles alunso que foram os dois dias

In [ ]:
vestibulandos_presentes['TP_PRESENCA_CN'].value_counts().sort_index()
# 583 eliminados

0       583
1    499940
2         3
Name: TP_PRESENCA_CN, dtype: int64

In [ ]:
vestibulandos_presentes['TP_PRESENCA_CH'].value_counts().sort_index()
# Compareceu mas foi eliminado

0         3
1    499940
2       583
Name: TP_PRESENCA_CH, dtype: int64

In [ ]:
vestibulandos_presentes['TP_PRESENCA_LC'].value_counts().sort_index()

0         3
1    499940
2       583
Name: TP_PRESENCA_LC, dtype: int64

In [ ]:
vestibulandos_presentes['TP_PRESENCA_MT'].value_counts().sort_index()
# 1 - Sem irregularidade
# 2 - Foi mas foi desclassificado
# 0 - Desclassificado

0       583
1    499940
2         3
Name: TP_PRESENCA_MT, dtype: int64

In [ ]:
vestibulandos_classificados = vestibulandos_presentes.loc[vestibulandos_presentes.TP_PRESENCA_CN == 1]

In [ ]:
vestibulandos_classificados['TP_PRESENCA_CN'].value_counts()
# Todos que não tiveram nenhum problema na prova

1    499940
Name: TP_PRESENCA_CN, dtype: int64

In [ ]:
vestibulandos_classificados['TP_PRESENCA_CH'].value_counts()
# Classificados

1    499940
Name: TP_PRESENCA_CH, dtype: int64

In [ ]:
vestibulandos_classificados['TP_PRESENCA_LC'].value_counts()

1    499940
Name: TP_PRESENCA_LC, dtype: int64

In [ ]:
vestibulandos_classificados['TP_PRESENCA_MT'].value_counts()

1    499940
Name: TP_PRESENCA_MT, dtype: int64

In [ ]:
provas = ['NOTA_REDACAO','NOTA_CN','NOTA_MT','NOTA_LC','NOTA_CH']

In [ ]:
vestibulandos_classificados[provas].isnull().sum()
# Eliminado os valores nulos

NOTA_REDACAO    0
NOTA_CN         0
NOTA_MT         0
NOTA_LC         0
NOTA_CH         0
dtype: int64

In [ ]:
# Notas zeros na Redação
vestibulandos_classificados.query('NOTA_REDACAO == 0')['NOTA_REDACAO'].value_counts()
# Contagem de notas zero

0.0    9181
Name: NOTA_REDACAO, dtype: int64

In [ ]:
vestibulandos_classificados.query('COMP1 == 0')['COMP1'].value_counts()
# Componentes 1 da redação / Quantos zeraram

0.0    9188
Name: COMP1, dtype: int64

In [ ]:
vestibulandos_classificados.query('COMP2 == 0')['COMP2'].value_counts()
# Componentes 2 da redação / Quantos zeraram

0.0    9181
Name: COMP2, dtype: int64

In [ ]:
vestibulandos_classificados.query('COMP3 == 0')['COMP3'].value_counts()
# Componentes 3 da redação / Quantos zeraram

0.0    9197
Name: COMP3, dtype: int64

In [ ]:
vestibulandos_classificados.query('COMP4 == 0')['COMP4'].value_counts()
# Componentes 4 da redação / Quantos zeraram

0.0    9194
Name: COMP4, dtype: int64

In [ ]:
vestibulandos_classificados.query('COMP5 == 0')['COMP5'].value_counts()
## Componentes 5 da redação / Quantos zeraram

0.0    73330
Name: COMP5, dtype: int64

# NOTAS ZEROS EM REDAÇÃO:

# REDAÇÃO (final): 9181 notas zeros devido a não cumprimento das regras ou tirou zero em todos os cinco componentes.

# COMPONENTE 1(Demonstrar domínio da modalidade escrita formal da Língua Portuguesa.): 9188 notas zeros somente neste item.

# COMPONENTE 2 (Compreender a proposta de redação e aplicar conceitos das várias áreas de conhecimento para desenvolver o tema, dentro dos limites estruturais do texto dissertativo-argumentativo em prosa): 9181.

# COMPONENTE 3 (Selecionar, relacionar, organizar e interpretar informações, fatos, opiniões e argumentos em defesa de um ponto de vista): 9197 notas zeros somente neste item.

# COMPONENTE 4 (Demonstrar conhecimento dos mecanismos linguísticos necessários para a construção da argumentação.): 9194 notas zeros somente neste item.

# COMPONENTE 5 (Elaborar proposta de intervenção para o problema abordado, respeitando os direitos humanos): 73330 notas zeros somente neste item.

In [ ]:
vestibulandos_classificados.query('NOTA_CH == 0')['NOTA_CH'].value_counts()
# Quantidade de alunos que zeraram nessa disciplina

0.0    343
Name: NOTA_CH, dtype: int64

In [ ]:
vestibulandos_classificados.query('NOTA_CN == 0')['NOTA_CN'].value_counts()
# Quantidade de alunos que zeraram nessa disciplina

0.0    36
Name: NOTA_CN, dtype: int64

In [ ]:
vestibulandos_classificados.query('NOTA_LC == 0')['NOTA_LC'].value_counts()
# Quantidade de alunos que zeraram nessa disciplina

0.0    135
Name: NOTA_LC, dtype: int64

In [ ]:
vestibulandos_classificados.query('NOTA_MT == 0')['NOTA_MT'].value_counts()
# Quantidade de alunos que zeraram nessa disciplina

0.0    56
Name: NOTA_MT, dtype: int64

In [ ]:
vestibulandos_classificados.query('NOTA_REDACAO == 0')['NOTA_REDACAO'].value_counts()
# Quantidade de alunos que zeraram nessa disciplina

0.0    9181
Name: NOTA_REDACAO, dtype: int64

# NOTAS ZEROS DOS INSCRITOS CONCORRENTES AO VESTIBULAR:

# Matemática = 56

# Ciências da Natureza = 36

# Linguagens e Códigos = 135

# Ciências Humanas = 343

# Redação = 9181

In [ ]:
vestibulandos_classificados.head()

NU_INSCRICAO NO_MUNICIPIO_RESIDENCIA  ...  INTERNET presenca_total
0     1,90E+11            Santa Isabel  ...       Sim              4
2     1,90E+11               São Paulo  ...       Não              4
3     1,90E+11   São Bernardo do Campo  ...       Sim              4
5     1,90E+11              Embu-Guaçu  ...       Sim              4
8     1,90E+11               Guarulhos  ...       Sim              4

[5 rows x 30 columns]

In [ ]:
vestibulandos_classificados = vestibulandos_classificados.drop(columns=['IN_TREINEIRO','presenca_total'], )

In [ ]:
vestibulandos_classificados.shape

(499940, 28)

In [ ]:
# Salvando (Exportando) o Dataframe Tratado
vestibulandos_classificados.to_csv('enem_2020_tratado.csv', encoding = 'iso-8859-1', index = False)